Makefile

```diff
@@ -159,7 +159,7 @@ terrascan:
 
 pre-pull:
 ifeq ($(TAG),devel)
-	@$(BUILDER) pull golang:1.18-bullseye
+	@$(BUILDER) pull golang:1.18.1-bullseye
```

Comment:
we haven't used the patch version in Dockerfiles so `golang:1.18-bullseye` is enough to get the latest.